.github/workflows/generate-benchmarks-s3.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         shell: bash
         run: |
           # We run each query once to make sure we don't upload a file if there's a bug that causes a panic.
-          cargo run --release --bin clickbench --package bench-vortex -- --targets datafusion:parquet,datafusion:vortex -i1
+          cargo run --release --bin query_bench --package bench-vortex -- clickbench --targets datafusion:parquet,datafusion:vortex -i1
```

Comment:
Minor thing but maybe nice as everyone's shell history has the old commands. We could construct different bin targets from the same files with diff names. This would also spare an extra arg on the cli. On the other hand, being able to pass subcommand specific flags is nice.